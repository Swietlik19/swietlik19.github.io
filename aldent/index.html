<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="UTF-8">
    <title> |</title>
    <meta name="description" content="">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="format-detection" content="telephone=no">
    <!-- Template Basic Images Start-->
    <!-- Template Basic Images End-->
    <!-- Custom Browsers Color Start-->
    <meta name="theme-color" content="#057bb2">
    <!-- Custom Browsers Color End-->
    <link rel="stylesheet" href="libs/normalize.min.css">
    <link rel="stylesheet" href="libs/animate.css">
    <link rel="stylesheet" href="css/first-load.css">
    <style>
      .mmenu__list {
        margin: 0;
        padding: 0;
        list-style: none;
        margin-top: 180px;
        margin-bottom: 40px;
      }

      .mmenu__item {
        padding: 10px 0;
      }

      .mmenu__link {
        text-decoration: none;
        color: #057bb2;
        -webkit-transition: 0.3s;
        -o-transition: 0.3s;
        transition: 0.3s;
        font-size: 24px;
      }

      .mmenu__link:hover {
        color: #8b0a0e;
      }

      @media screen and (max-width: 768px) {
        .mmenu__list {
          margin-top: 100px;
        }
      }
    </style>
  </head>
  <body class="main-page">
    <header class="header" id="header">
      <div class="header__top">
        <div class="container">
          <div class="header__wrap">
            <div class="logo__wrap">
              <div class="logo"><img src="img/svg/logo.svg" alt="лого"></div>
            </div>
            <nav class="header__nav">
              <div class="main-menu">
                <button class="burger" id="header-burger"><span class="burger__wrap"><span class="line"></span></span></button>
                <ul class="menu">
                  <li class="menu-item"><a href="#">О клинике</a></li>
                  <li class="menu-item"><a href="#">Услуги</a></li>
                  <li class="menu-item"><a href="#">Цены</a></li>
                  <li class="menu-item"><a href="#">Отзывы</a></li>
                  <li class="menu-item"><a href="#">Новости</a></li>
                  <li class="menu-item"><a href="#">Контакты</a></li>
                </ul>
              </div>
              <!-- /.container-->
            </nav>
            <!-- /.header__nav-->
            <div class="header__info">
              <div class="header__phones"><a class="btn btn--circle open-modal desktop" href="#modal__callback"><i class="fa fa-phone"></i></a><a class="btn btn--circle mobile" href="tel:+79780241220"><i class="fa fa-phone"></i></a></div>
              <div class="header__content"><a class="header__link" href="tel:+79780241220">+7 (978) 024-12-20</a>
                <div class="header__item">
                  <p>ПН-ПТ: 9:00-20:00,<br> СБ до 14:00, ВС - выходной</p>
                </div>
                <div class="header__item">
                  <p>г. Симферополь,<br> ул. Железнодорожная, 1 В</p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <!-- /.container-->
      </div>
      <!-- /.header__top-->
    </header>
    <div class="page_wr" id="page_wr">
      <section class="mmenu">
        <div class="container">
          <ul class="mmenu__list">
            <li class="mmenu__item"><a class="mmenu__link" href="main.html">Главная</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="blog.html">Блог</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="post.html">Статья</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="contacts.html">Контакты</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="about.html">О компании</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="reviews.html">Отзывы</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="services.html">Услуги</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="prices.html">Цены</a></li>
          </ul>
        </div>
      </section>
    </div>
    <footer class="footer" id="footer"><a class="to-top" href="#page_wr"><i class="fa fa-angle-up"></i></a>
      <div class="footer__top">
        <div class="container">
          <div class="footer__df">
            <div class="footer__content">
              <div class="logo__wrap">
                <div class="logo"><img src="img/svg/logo1.svg" alt="Лого"></div>
              </div>
              <div class="footer__address">
                <div class="footer__item">
                  <p class="footer__text">г. Симферополь,<br> ул. Железнодорожная, 1В</p>
                </div>
                <div class="footer__item">
                  <p class="footer__text">Без выходных: 9:00-20:00</p>
                </div><a class="footer__tel" href="tel:+79780241220">+7 (978) 024-12-20</a>
                <section class="socials" id="socials"><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-vk"></i></a><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-facebook"></i></a><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-instagram"></i></a></section>
                <!-- /.socials-->
              </div>
            </div>
            <div class="footer__menu">
              <ul class="menu">
                <li class="menu-item menu-item-has-children"><a href="#">Услуги</a>
                  <button class="menu-arrow"></button>
                  <ul class="sub-menu">
                    <li class="menu-item"><a href="#">Терапия</a></li>
                    <li class="menu-item"><a href="#">Хирургия</a></li>
                    <li class="menu-item"><a href="#">Ортопедия</a></li>
                    <li class="menu-item"><a href="#">Ортодонтия</a></li>
                  </ul>
                </li>
              </ul>
            </div>
            <div class="footer__menu">
              <ul class="menu">
                <li class="menu-item"><a href="#">О клинике</a></li>
                <li class="menu-item"><a href="#">Цены</a></li>
                <li class="menu-item"><a href="#">Отзывы</a></li>
                <li class="menu-item"><a href="#">Новости</a></li>
                <li class="menu-item"><a href="#">Контакты</a></li>
              </ul>
            </div>
          </div>
        </div>
        <!-- /.container-->
      </div>
      <!-- /.footer__top-->
      <div class="footer__bottom">
        <div class="container">
          <div class="row">
            <div class="col-xl-3 col-lg-4 col-sm-7">
              <div class="copyright">
                <p class="text">г. Симферополь © Copyright 2019&nbsp;г.</p>
              </div>
            </div>
            <div class="col-xl-4 offset-xl-1 col-lg-4 col-sm-5"><a class="made" href="https://sheer82.ru/"><span class="text">Сделано в</span>
                <svg class="icon">
                  <use xlink:href="img/svg/sprite.svg#iconSheer"></use>
                </svg></a></div>
          </div>
          <!-- /.row-->
        </div>
        <!-- /.container-->
      </div>
      <!-- /.footer__bottom-->
    </footer>
    <!-- /.footer-->
    <link rel="stylesheet" href="libs/nice-select/nice-select.min.css">
    <link rel="stylesheet" href="css/main.css">
    <script src="js//libs/jquery.min.js"></script>
    <script src="js/libs/jquery.lazyload.min.js"></script>
    <script src="js/libs/wow.min.js"></script>
    <script src="js/libs/jquery.maskedinput.min.js"></script>
    <script src="js/libs/slick.min.js"></script>
    <script src="js/libs/svg4everybody.min.js"></script>
    <script src="js/libs/ofi.min.js"></script>
    <script src="libs/fancybox/jquery.fancybox.pack.js"></script>
    <script src="libs/fancybox/helpers/jquery.fancybox-buttons.js"></script>
    <script src="libs/nice-select/jquery.nice-select.min.js"></script>
    <script src="js/common.js"></script>
    <script src="js/slider.js"></script>
  </body>
</html>