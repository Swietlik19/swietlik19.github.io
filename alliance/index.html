<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="UTF-8">
    <title> |</title>
    <meta name="description" content="">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="format-detection" content="telephone=no">
    <!-- Template Basic Images Start-->
    <!-- Template Basic Images End-->
    <!-- Custom Browsers Color Start-->
    <meta name="theme-color" content="#432928">
    <!-- Custom Browsers Color End-->
    <style>
      .mmenu__list {
        margin: 0;
        padding: 0;
        list-style: none;
        margin-top: 40px;
        margin-bottom: 40px;
      }

      .mmenu__item {
        padding: 10px 0;
      }

      .mmenu__link {
        text-decoration: none;
        color: #432928;
        -webkit-transition: 0.3s;
        -o-transition: 0.3s;
        transition: 0.3s;
        font-size: 24px;
      }

      .mmenu__link:hover {
        color: #8b0a0e;
      }
    </style>
  </head>
  <body class="main-page">
    <header class="header" id="header">
      <div class="header__wrap">
        <div class="logo__wrap">
          <div class="logo"><img src="img/svg/logo.svg" alt="лого"></div>
        </div>
        <button class="burger" id="header__burger"><span class="burger__wrap"><span class="line"></span></span></button>
        <section class="socials" id="socials"><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-vk"></i></a><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-facebook"></i></a><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-instagram"></i></a></section>
        <!-- /.socials-->
        <div class="header__phones">
          <div class="header__phone desktop"><a href="tel:+79780293010" rel="nofollow">+7 (978) 029-30-10</a></div>
          <div class="header__phone mobile"><a class="btn btn--circle" href="tel:+79780293010" rel="nofollow"><i class="fa fa-phone"></i></a></div>
        </div>
      </div>
      <!-- /.header__wrap-->
    </header>
    <div class="page_wr" id="page_wr">
      <section class="mmenu">
        <div class="container">
          <ul class="mmenu__list">
            <li class="mmenu__item"><a class="mmenu__link" href="main.html">Главная</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="about.html">О компании</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="blog.html">Блог</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="post.html">Статья</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="services.html">Сервисы</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="contacts.html">Контакты</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="equip.html">Оснащение</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="proj.html">Проекты</a></li>
            <li class="mmenu__item"><a class="mmenu__link" href="serv-item.html">Одна услуга</a></li>
          </ul>
        </div>
      </section>
    </div>
    <footer class="footer" id="footer">
      <div class="container">
        <div class="footer__df">
          <div class="footer__info">
            <div class="footer__text footer__text--phone"><a href="tel:+79780293010" rel="nofollow">+7 (978) 029-30-10</a></div>
            <div class="footer__text"><a href="mailto:test@mail.ru" rel="nofollow">test@mail.ru</a></div>
          </div>
          <div class="footer__address">
            <div class="footer__text">
              <p>г. Симферополь, пр. Кирова, 29/1, оф., 412<br> Пн-Пт с 10:00 до 19:00</p>
            </div>
          </div>
          <section class="socials" id="socials-footer"><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-vk"></i></a><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-facebook"></i></a><a class="socials__link" href="#" rel="nofollow" target="_blank"><i class="fa fa-instagram"></i></a></section>
          <!-- /.socials-->
          <div class="footer__politics">
            <div class="footer__text"><a href="#">Политика конфиденциальности</a></div>
          </div><a class="made" href="https://sheer82.ru/"><span class="footer__text">Сделано в</span>
            <svg class="icon">
              <use xlink:href="img/svg/sprite.svg#iconSheer"></use>
            </svg></a>
        </div>
      </div>
      <!-- /.footer__df-->
    </footer>
    <!-- /.footer-->
    <link rel="stylesheet" href="libs/normalize.min.css">
    <link rel="stylesheet" href="libs/animate.css">
    <link rel="stylesheet" href="libs/swiper/swiper.min.css">
    <link rel="stylesheet" href="libs/fullpage/fullpage.min.css">
    <link rel="stylesheet" href="libs/mcsb/jquery.mCustomScrollbar.min.css">
    <link rel="stylesheet" href="css/main.css">
    <script src="js//libs/jquery.min.js"></script>
    <script src="js/libs/jquery.lazyload.min.js"></script>
    <script src="js/libs/wow.min.js"></script>
    <script src="js/libs/jquery.maskedinput.min.js"></script>
    <script src="js/libs/svg4everybody.min.js"></script>
    <script src="js/libs/ofi.min.js"></script>
    <script src="libs/fancybox/jquery.fancybox.pack.js"></script>
    <script src="libs/fancybox/helpers/jquery.fancybox-buttons.js"></script>
    <script src="libs/swiper/swiper.js"></script>
    <script src="libs/fullpage/jquery.fullpage.js"></script>
    <script src="libs/mcsb/jquery.mCustomScrollbar.concat.min.js"></script>
    <script src="js/common.js"></script>
  </body>
</html>
