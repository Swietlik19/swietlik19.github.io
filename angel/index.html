<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="UTF-8">
    <title> |</title>
    <meta name="description" content="">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="format-detection" content="telephone=no">
    <!-- Template Basic Images Start-->
    <!-- Template Basic Images End-->
    <!-- Custom Browsers Color Start-->
    <meta name="theme-color" content="#432928">
    <!-- Custom Browsers Color End-->
    <style>
      .mmenu__list
      {
        margin: 0;
        padding: 0;
        list-style: none;
        margin-top: 40px;
        margin-bottom: 40px;
      }

      .mmenu__item
      {
        padding: 10px 0;
      }

      .mmenu__link
      {
        text-decoration: none;
        color: #432928;
        -webkit-transition: 0.3s;
        -o-transition: 0.3s;
        transition: 0.3s;
        font-size: 24px;
      }

      .mmenu__link:hover
      {
        color: #8b0a0e;
      }
    </style>
    <link rel="stylesheet" href="libs/normalize.min.css">
    <link rel="stylesheet" href="libs/animate.css">
    <link rel="stylesheet" href="libs/swiper/swiper.min.css">
    <link rel="stylesheet" href="css/main.css">
  </head>
  <body class="main-page">
    <header class="header" id="header">
      <div class="header__wrap">
        <div class="wr">
          <div class="header__top">
            <div class="logo__wrap">
              <div class="logo"><img src="img/logo.png" alt="лого"></div>
              <div class="logo__content">
                <p><strong>Ангел Строитель</strong><br><span>Строительство и проектирование<br> домов под ключ в Крыму</span></p>
              </div>
            </div>
            <div class="header__address">
              <p>Севастополь, ул. Дм. Ульянова, 1а<a href="mailto:krmstroypost@gmail.com" rel="nofollow">krmstroypost@gmail.com</a></p>
            </div>
            <div class="header__phones">
              <div class="header__phones-left"><a class="header__phone desktop" href="tel:+73652694040" rel="nofollow">+7 (3652) 694-040</a><a class="xbtn xbtn--circle mobile" href="tel:+73652694040" rel="nofollow">
                  <svg class="icon">
                    <use xlink:href="img/svg/sprite.svg#phone-call"></use>
                  </svg></a><a class="header__phone desktop" href="tel:+79788005468" rel="nofollow">+7 (978) 800-54-68</a></div><a class="xbtn xbtn--circle open-modal" href="#modal_callback">
                <svg class="icon">
                  <use xlink:href="img/svg/sprite.svg#phone-call"></use>
                </svg></a>
            </div>
          </div>
          <!-- /.header__top-->
        </div>
        <!-- /.wr-->
      </div>
      <div class="header__bottom">
        <div class="wr">
          <nav>
            <button class="burger"><span class="line"></span></button>
            <ul class="menu">
              <li class="menu-item"><a href="#">Услуги</a></li>
              <li class="menu-item"><a href="#">Портфолио</a></li>
              <li class="menu-item"><a href="#">Купить дом</a></li>
              <li class="menu-item"><a href="#">О компании</a></li>
              <li class="menu-item"><a href="#">Блог</a></li>
              <li class="menu-item"><a href="#">Контакты</a></li>
            </ul>
          </nav>
          <!-- /nav-->
        </div>
        <!-- /.wr-->
      </div>
      <!-- /.header__bottom-->
    </header>
    <div class="page_wr" id="page_wr">
      <section class="mmenu">
        <div class="container">
          <ul class="mmenu__list">
            <li class="mmenu__item"><a class="mmenu__link" href="main.html">Главная</a></li>
          </ul>
        </div>
      </section>
    </div>
    <footer class="footer" id="footer">
      <div class="wr">
        <div class="footer__top">
          <div class="footer__col">
            <div class="logo__wrap">
              <div class="logo"><img src="img/logo.png" alt="лого"></div>
            </div>
            <div class="footer__content">
              <div class="logo__text">
                <p> <strong>Ангел Строитель</strong> Строительство и проектирование домов под ключ в Крыму</p>
              </div>
              <section class="socials socials--footer" id="socials-footer"><a class="socials__link" href="#" rel="nofollow" target="_blank">
                  <svg class="icon">
                    <use xlink:href="img/svg/sprite.svg#vk"></use>
                  </svg></a><a class="socials__link" href="#" rel="nofollow" target="_blank">
                  <svg class="icon">
                    <use xlink:href="img/svg/sprite.svg#insta"></use>
                  </svg></a><a class="socials__link" href="#" rel="nofollow" target="_blank">
                  <svg class="icon">
                    <use xlink:href="img/svg/sprite.svg#yt"></use>
                  </svg></a></section>
              <!-- /.socials-->
            </div>
          </div>
          <!-- /.footer__col-->
          <div class="footer__col">
            <h3 class="footer__title">Наши услуги</h3>
            <div class="footer__hidden">
              <ul class="menu">
                <li class="menu-item"><a href="#">Строительство</a></li>
                <li> <a href="#">Индивид. проектирование</a></li>
                <li> <a href="#">Готовые проекты домов</a></li>
                <li> <a href="#">Поиск и проверка участков</a></li>
              </ul>
            </div>
          </div>
          <!-- /.footer__col-->
          <div class="footer__col">
            <div class="footer__main">
              <ul class="menu">
                <li class="menu-item"><a href="#">Портфолио</a></li>
                <li class="menu-item"><a href="#">Купить дом</a></li>
                <li class="menu-item"><a href="#">О компании</a></li>
                <li class="menu-item"><a href="#">Вопрос-Ответ</a></li>
                <li class="menu-item"><a href="#">Блог</a></li>
              </ul>
            </div>
          </div>
          <!-- /.footer__col-->
          <div class="footer__col">
            <h3 class="footer__title">Контакты</h3>
            <div class="footer__item">
              <h4>Адрес:</h4>
              <p>АР Крым, Севастополь<br> ул. Дмитрия Ульянова 1а</p>
            </div>
            <div class="footer__item">
              <h4>Телефон:</h4><a href="tel:+79788005468" rel="nofollow">+7 978 800-54-68</a>
            </div>
            <div class="footer__item">
              <h4>E-mail:</h4><a href="mailto:krmstroypost@gmail.com" rel="nofollow">krmstroypost@gmail.com</a>
            </div>
          </div>
          <!-- /.footer__col-->
        </div>
        <!-- /.footer__top-->
      </div>
      <!-- /.container-->
      <div class="footer__bottom">
        <div class="wr">
          <div class="footer__bottom-df">
            <div class="footer__politics"><a class="text" href="#">Политика конфиденциальности</a></div>
            <div class="footer__copyright">
              <p class="text">г. Симферополь © Copyright 2021&nbsp;г.</p>
            </div><a class="made" href="https://sheer82.ru/"><span class="text">Сделано в</span>
              <svg class="icon">
                <use xlink:href="img/svg/sprite.svg#iconSheer"></use>
              </svg></a>
          </div>
          <!-- /.row-->
        </div>
        <!-- /.container-->
      </div>
      <!-- /.footer__bottom-->
    </footer>
    <!-- /.footer-->
    <script src="js//libs/jquery.min.js"></script>
    <script src="js/libs/jquery.maskedinput.min.js"></script>
    <script src="js/libs/svg4everybody.min.js"></script>
    <script src="js/libs/ofi.min.js"></script>
    <script src="libs/fancybox/jquery.fancybox.pack.js"></script>
    <script src="libs/fancybox/helpers/jquery.fancybox-buttons.js"></script>
    <script src="libs/swiper/swiper.js"></script>
    <script src="js/common.js"></script>
  </body>
</html>