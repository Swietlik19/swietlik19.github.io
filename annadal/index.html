<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="UTF-8">
    <title> |</title>
    <meta name="description" content="">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="format-detection" content="telephone=no">
    <!-- Template Basic Images Start-->
    <!-- Template Basic Images End-->
    <!-- Custom Browsers Color Start-->
    <meta name="theme-color" content="#432928">
    <!-- Custom Browsers Color End-->
    <style>
      .mmenu__list {
        margin: 0;
        padding: 0;
        list-style: none;
        margin-top: 100px;
        margin-bottom: 40px;
      }

      .mmenu__item {
        padding: 10px 0;
      }

      .mmenu__link {
        text-decoration: none;
        color: #432928;
        -webkit-transition: 0.3s;
        -o-transition: 0.3s;
        transition: 0.3s;
        font-size: 24px;
      }

      .mmenu__link:hover {
        color: #8b0a0e;
      }
    </style>
  </head>
  <body class="main-page">
    <header class="header" id="header">
      <div class="container">
        <div class="header__top">
          <div class="header__phones">
            <div class="header__phone"><a class="mobile" href="tel:+79167034040" rel="nofollow"><span class="btn btn--circle"><i class="fa fa-phone"></i></span></a><a class="desktop" href="tel:+79167034040" rel="nofollow">+7 916<strong> 703 40 40</strong><small> (Москва)</small></a></div>
            <div class="header__phone"><a class="desktop" href="tel:+79781093015" rel="nofollow">+7 978<strong> 109 30 15</strong><small> (Симферополь)</small></a></div>
          </div>
          <div class="logo__wrap">
            <div class="logo"><img src="img/svg/logo.svg" alt="лого"></div>
            <p class="logo__text">Врач косметолог, трихолог</p>
          </div>
          <div class="header__burger">
            <button class="burger"><span class="burger__wrap"><span class="line"></span></span></button>
          </div>
        </div>
        <!-- /.header__top-->
      </div>
      <!-- /.container-->
      <div class="header__menu">
        <div class="container">
          <div class="header__menu-wrap">
            <div class="logo__wrap">
              <div class="logo"><img src="img/svg/logo-menu.svg" alt="лого"></div>
            </div>
            <ul class="menu">
              <li class="menu-item"><a class="inner-link" href="#scr_serv">Какие проблемы решаю</a></li>
              <li class="menu-item"><a class="inner-link" href="#scr_about">Обо мне</a></li>
              <li class="menu-item"><a class="inner-link" href="#scr_procedures">Услуги</a></li>
              <li class="menu-item"><a class="inner-link" href="#scr_cases">Кейсы</a></li>
              <li class="menu-item"><a class="inner-link" href="#scr_reviews">Отзывы</a></li>
              <li class="menu-item"><a class="inner-link" href="#scr_contacts">Контакты</a></li>
            </ul>
          </div>
          <!-- /.header__menu-wrap-->
        </div>
        <!-- /.container-->
      </div>
      <!-- /.header__menu-->
    </header>
    <div class="page_wr" id="page_wr">
      <section class="mmenu">
        <div class="container">
          <ul class="mmenu__list">
            <li class="mmenu__item"><a class="mmenu__link" href="main.html">Главная</a></li>
          </ul>
        </div>
      </section>
    </div>
    <footer class="footer" id="footer">
      <div class="footer__top">
        <div class="container">
          <ul class="menu">
            <li class="menu-item"><a class="inner-link" href="#scr_serv">Какие проблемы решаю</a></li>
            <li class="menu-item"><a class="inner-link" href="#scr_about">Обо мне</a></li>
            <li class="menu-item"><a class="inner-link" href="#scr_procedures">Услуги</a></li>
            <li class="menu-item"><a class="inner-link" href="#scr_cases">Кейсы</a></li>
            <li class="menu-item"><a class="inner-link" href="#scr_reviews">Отзывы</a></li>
            <li class="menu-item"><a class="inner-link" href="#scr_contacts">Контакты</a></li>
          </ul>
        </div>
        <!-- /.container-->
      </div>
      <!-- /.footer__top-->
      <div class="footer__bottom">
        <div class="container">
          <p>имеются противопоказания. проконсультируйтесь со специалистом</p>
        </div>
        <!-- /.container-->
      </div>
      <!-- /.footer__bottom-->
    </footer>
    <!-- /.footer-->
    <link rel="stylesheet" href="libs/normalize.min.css">
    <link rel="stylesheet" href="libs/animate.css">
    <link rel="stylesheet" href="libs/swiper/swiper.min.css">
    <link rel="stylesheet" href="libs/twenty/twentytwenty.css">
    <link rel="stylesheet" href="css/main.css">
    <script src="js//libs/jquery.min.js"></script>
    <script src="js/libs/jquery.lazyload.min.js"></script>
    <script src="js/libs/wow.min.js"></script>
    <script src="js/libs/jquery.maskedinput.min.js"></script>
    <script src="js/libs/svg4everybody.min.js"></script>
    <script src="js/libs/ofi.min.js"></script>
    <script src="js/libs/nums.js"></script>
    <script src="libs/fancybox/jquery.fancybox.pack.js"></script>
    <script src="libs/fancybox/helpers/jquery.fancybox-buttons.js"></script>
    <script src="libs/swiper/swiper.js"></script>
    <script src="libs/twenty/jquery.event.move.js"></script>
    <script src="libs/twenty/jquery.twentytwenty.js"></script>
    <script src="js/common.js"></script>
  </body>
</html>