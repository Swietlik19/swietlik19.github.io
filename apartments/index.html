<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="UTF-8">
    <title>Квартиры</title>
    <meta name="description" content="">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="format-detection" content="telephone=no">
    <!-- Custom Browsers Color Start-->
    <meta name="theme-color" content="#0B1739">
    <!-- Custom Browsers Color End-->
    <link rel="stylesheet" href="libs/normalize.min.css">
    <link rel="stylesheet" href="libs/fancybox/jquery.fancybox.min.css">
    <link rel="stylesheet" href="libs/jRange/jquery.range.css">
    <link rel="stylesheet" href="css/main.css">
  </head>
  <body>
    <div class="page_wr" id="page_wr">
      <section class="scr1" id="scr1">
        <div class="wr">

          <div class="scr1__df">
            <div class="scr1__apartments">
              <h1 class="page-title">Квартиры</h1>
              <div class="scr1__wrap">

                <div class="scr1__row scr1__row--sort">
                  <div class="scr1__col"><span>Планировка</span></div>
                  <div class="scr1__col"><span>Квартира</span></div>
                  <div class="scr1__col">
                    <button class="scr1__sort" type="button" tabindex="0" data-sort="square">S, м<sup>²</sup>
                      <span class="scr1__sort-arrow scr1__sort-arrow--asc">
                        <svg class="icon">
                          <use xlink:href="img/svg/sprite.svg#arrow-sort"></use>
                        </svg>
                      </span>
                      <span class="scr1__sort-arrow scr1__sort-arrow--desc">
                        <svg class="icon">
                          <use xlink:href="img/svg/sprite.svg#arrow-sort"></use>
                        </svg>
                      </span>
                    </button>
                  </div>

                  <div class="scr1__col">
                    <button class="scr1__sort" type="button" tabindex="0" data-sort="stage">Этаж
                      <span class="scr1__sort-arrow scr1__sort-arrow--asc">
                        <svg class="icon">
                          <use xlink:href="img/svg/sprite.svg#arrow-sort"></use>
                        </svg>
                      </span>
                      <span class="scr1__sort-arrow scr1__sort-arrow--desc">
                        <svg class="icon">
                          <use xlink:href="img/svg/sprite.svg#arrow-sort"></use>
                        </svg>
                      </span>
                    </button>
                  </div>

                  <div class="scr1__col">
                    <button class="scr1__sort scr1__sort--active scr1__sort--asc" type="button" tabindex="0" data-sort="price">Цена, ₽
                      <span class="scr1__sort-arrow scr1__sort-arrow--asc">
                        <svg class="icon">
                          <use xlink:href="img/svg/sprite.svg#arrow-sort"></use>
                        </svg>
                      </span>
                      <span class="scr1__sort-arrow scr1__sort-arrow--desc">
                        <svg class="icon">
                          <use xlink:href="img/svg/sprite.svg#arrow-sort"></use>
                        </svg>
                      </span>
                    </button>
                  </div>
                </div>
                <div class="scr1__content"></div>
              </div>

              <div class="scr1__btns">
                <div class="btn__wrap">
                  <a class="xbtn js-load-more" href="#" data-count="20">Загрузить еще</a>
                </div>
              </div>
            </div>

            <aside class="scr1__filters">
              <form>
                <div class="scr1__filters-rooms">
                  <div class="scr1__filters-rooms-item">
                    <label>
                      <input type="checkbox" name="rooms" value="1"><span>1к</span>
                    </label>
                  </div>
                  <div class="scr1__filters-rooms-item">
                    <label>
                      <input type="checkbox" name="rooms" value="2" checked><span>2к</span>
                    </label>
                  </div>
                  <div class="scr1__filters-rooms-item">
                    <label>
                      <input type="checkbox" name="rooms" value="3"><span>3к</span>
                    </label>
                  </div>
                  <div class="scr1__filters-rooms-item">
                    <label>
                      <input type="checkbox" name="rooms" value="4" disabled><span>4к</span>
                    </label>
                  </div>
                </div>

                <div class="scr1__filters-range">
                  <div class="scr1__range">
                    <label>Стомость квартиры, ₽ <input class="wpcf7-form-control wpcf7-text wpcf7-count scr1__range-item scr1__range-item--price" type="text" name="range-price" aria-required="true" aria-invalid="false" data-min="5500000" data-max="18900000" value="5500000,18900000">
                    </label>
                  </div>
                </div>

                <div class="scr1__filters-range">
                  <div class="scr1__range">
                    <label>Площадь, м<sup>²</sup>
                      <input class="wpcf7-form-control wpcf7-text wpcf7-count scr1__range-item scr1__range-item--square" type="hidden" name="range-price" aria-required="true" aria-invalid="false" data-min="33" data-max="123" value="33,123">
                    </label>
                  </div>
                </div>

                <div class="scr1__filters-btns">
                  <button class="js-btn-reset" type="reset">Сбросить параметры</button>
                </div>

              </form>
            </aside>
          </div>
        </div>
        <!-- /.wr-->
      </section>
      <!-- /.scr1-->

      <div class="to-top"></div>

    </div>

    <script src="libs/jquery.min.js"></script>
    <script src="libs/fancybox/jquery.fancybox.min.js"></script>
    <script src="libs/svg4everybody.min.js"></script>
    <script src="js/common.js"></script>
  </body>
</html>
