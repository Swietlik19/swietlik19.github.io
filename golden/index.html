<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Animal clinic</title>
    <link href="css/style.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
  </head>
  <body>
    <header class="main-header">
      <div class="main-header__container">
        <div class="main-header__top">
          <div class="main-header__logo">
            <img src="img/logo.png" width="31" height="45">
          </div>
          <h1 class="page-title">Gabinet Weterynaryjny</h1>
        </div>
        <nav class="main-nav">
          <ul class="main-nav__list">
            <li class="main-nav__list-item">Home</li>
            <li class="main-nav__list-item">O nas</li>
            <li class="main-nav__list-item">Gabinet</li>
            <li class="main-nav__list-item">Godziny</li>
            <li class="main-nav__list-item">Kontakt</li>
          </ul>
        </nav>
      </div>
    </header>

     <main class="main-content">
        <div class="main-content__container">
          <p>Phasellus ultrices viverra tellus eget hendrerit. Pellentesque ligula metus, ullamcorper sit amet eleifend ut, mollis vel orci. In in ante sed nibh mollis pretium. Mauris sodales elementum nunc in molestie. Duis dapibus rhoncus ligula eget euismod. Aliquam erat volutpat. Fusce nibh erat, porttitor vel congue id, imperdiet ac magna. In auctor pulvinar vestibulum. Maecenas vestibulum diam a neque posuere a consectetur diam luctus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae.</p>
        </div>
      </main>


    <footer class="main-footer">

    </footer>

  </body>
</html>
